jupyter/laceworkjupyter/plugins/__init__.py

```diff
@@ -1,11 +1,13 @@
 """A simple loading of plugins."""
 
+from . import alerts
 from . import alert_rules
 from . import datasources
 
 
 PLUGINS = {
+    'alerts.get': alerts.process_alerts,
     'alert_rules.get': alert_rules.process_alert_rules,
     'datasources.list_data_sources': datasources.process_list_data_sources,
-    'datasources.get_datasource_schema': datasources.process_datasource_schema,
+    'datasources.get_datasource': datasources.process_datasource_schema,
```

Comment:
Sorry about this...  would it make sense to pin the laceworksdk version to the major version to prevent this in the future?  So, `laceworksdk~=1.0` moving forward?  I try to follow semantic versioning for that reason.

Reply:
yes, that's something we should do, if we are making larger changes, for now this is probably fine, but that's something that we shold do at some point